DataConnectors/Citrix_ADC_WAF.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id": "Citrix",
+    "title": "Citrix ADC WAF",
+    "publisher": "Citrix System",
+    "descriptionMarkdown": "Common Event Format (CEF) is an industry standard format on top of Syslog messages, used by many security vendors to allow event interoperability among different platforms. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, and threat intelligence enrichment for each log.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CitrixWafLogs",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Citrix\"\n| where DeviceProduct == \"NetScaler\"\n"
```

Comment:
Are there other logs that can share the same DeviceProduct name? To ensure this is uniquely recognizable later on would be great to name the DeviceProduct as "NetscalerWAF" - please reflect this change everywhere (like sample queries and other connectovity queries, etc.), if you change the DeviceProduct name. That ways if later you bring in Netscaler other logs (non-Waf) those can have the right DeviceProduct as well. That way it can be determined if this data source is connected properly.

Reply:
Currently there are no other logs with same DeviceProduct name. When adding support for CEF logs in future for different modules, we will make it more unique for that module. Renaming DeviceProduct now might not be feasible, as our customers might be using this already. So for now we want to use the same DeviceProduct.